cosmicfishpie/configs/config.py

```diff
@@ -207,6 +206,10 @@ def init(
     settings.setdefault("survey_specs", "ISTF-Optimistic")
     settings.setdefault("survey_name_photo", "Euclid-Photometric-ISTF-Pessimistic")
     settings.setdefault("survey_name_spectro", "Euclid-Spectroscopic-ISTF-Pessimistic")
+    settings.setdefault("survey_name_radio_photo", "SKA1-Photometric-Redbook-Optimistic")
```

Comment:
in theory we should add this also to the options part above.

Reply:
all this is fixed in the new commits!